commons/src/main/java/io/aiven/kafka/connect/common/source/input/Transformer.java

```diff
@@ -57,7 +58,9 @@ public final Stream<T> getRecords(final IOSupplier<InputStream> inputStreamIOSup
     protected abstract StreamSpliterator<T> createSpliterator(IOSupplier<InputStream> inputStreamIOSupplier,
             String topic, int topicPartition, AbstractConfig sourceConfig);
 
-    public abstract byte[] getValueBytes(T record, String topic, AbstractConfig sourceConfig);
+    public abstract SchemaAndValue getValueData(T record, String topic, AbstractConfig sourceConfig);
+
+    public abstract SchemaAndValue getKeyData(Object record, String topic, AbstractConfig sourceConfig);
```

Comment:
:D I'm sorry for this very late request, but it's small!

Can we change the parameter to this method throughout this PR from `record` to something like `cloudStorageKey` or `objectKey` ?  When I look at Avro (for example), the implication is that the first parameter to the `getValueData` and `getKeyData` would be the same object, when they really aren't.

Reply:
Indeed they are different, great suggestion. Updated.